var/spack/repos/builtin/packages/py-httpstan/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyHttpstan(PythonPackage):
+    """HTTP-based REST interface to Stan, a package for Bayesian inference."""
+
+    homepage = "https://mc-stan.org/"
+    url      = "https://github.com/stan-dev/httpstan/archive/refs/tags/4.6.1.tar.gz"
+
+    maintainers = ['haralmha']
+
+    version('4.6.1', sha256='703e5e04e60651e0004574bb9695827d759fd13eb0d6bd67f827c1bfa0a1fd31')
+
+    depends_on('python@3.7:3', type=('build', 'run'))
+    depends_on('py-setuptools@41.0:', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@41.0:', type=('build', 'run'))
```
`setuptools` is imported at run-time